.code-samples.meilisearch.yaml

```diff
@@ -74,6 +74,9 @@ search_get_1: |-
 get_update_1: |-
   curl \
     -X GET 'http://localhost:7700/indexes/movies/updates/1'
+get_update_2: |-
```

Comment:
Could we change the code sample name and position in `.code-samples.meilisearch.yaml` to reflect that it's used in the quick start guide? Something like `getting_started_get_update` would work. The current name + location make it seem like it's used in the API references.